ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/institutionServiceAgent.service.ts

```diff
@@ -18,11 +18,15 @@ export class InstitutionServiceAgent extends SessionServiceAgent<any> {
         super(session, configuration, http, logger, null);
     }
 
-    public getSummary(input: Observable<string>): Observable<Institution[]> {
+    public getSummary(input: Observable<string>, limit?: number): Observable<Institution[]> {
         return input.pipe(
-            flatMap(item =>
-                this.getObservable(`/autocomplete/institution?namePattern=${item}`, null),
```

Comment:
This is technically a bug. Putting the pattern into the URL using string interpolation will not properly do URL encoding. Instead, we should pass it in as query parameters.